eng/testing/performance/microbenchmarks.proj

```diff
@@ -131,12 +131,13 @@
     <HelixWorkItem Include="@(Partition)">
       <PayloadDirectory>$(WorkItemDirectory)</PayloadDirectory>
       <PreCommands Condition="'$(Compare)' == 'true'">$(WorkItemCommand) --bdn-artifacts $(BaselineArtifactsDirectory) --bdn-arguments="--anyCategories $(BDNCategories) $(ExtraBenchmarkDotNetArguments) $(BaselineCoreRunArgument) --partition-count $(PartitionCount) --partition-index %(HelixWorkItem.Index)"</PreCommands>
-      <Command>
-        if [ "x$PERF_PREREQS_INSTALLED" = "x1" ]; then
+      <Command Condition="'$(AGENT_OS)' != 'Windows_NT'">
+        if [[ "x$PERF_PREREQS_INSTALLED" = "x1" ]]; then
           $(WorkItemCommand) --bdn-artifacts $(ArtifactsDirectory) --bdn-arguments="--anyCategories $(BDNCategories) $(ExtraBenchmarkDotNetArguments) $(CoreRunArgument) --partition-count $(PartitionCount) --partition-index %(HelixWorkItem.Index)";
```

Comment:
The main command string can be moved to a property, without the `--partition-index %(HelixWorkItem.Index)`. then in `Command` metadata, we can add that index.

Reply:
I am not 100% sure what you are thinking. If you can make a suggestion, or open a different PR with these changes and the ones you are thinking I can review it and close this one.

Reply:
This is fine too 👍 The alternative gets unnecessarily complex.